e2etests/go.mod

```diff
@@ -6,21 +6,21 @@ toolchain go1.22.4
 
 replace (
 	github.com/metallb/frr-k8s => ../
-	go.universe.tf/e2etest => github.com/metallb/metallb/e2etest v0.0.0-20240715121012-af3c10d65f18
+	go.universe.tf/e2etest => github.com/karampok/metallb/e2etest v0.0.0-20240613071516-2d52178fbc89
```

Comment:
to be removed?

Reply:
Once this https://github.com/metallb/frr-k8s/pull/239 is merged, I would refactor and this PR should not have any changes in the go.mod.

I am drafting the PR until the dependency is merged